src/DetailsView/components/change-assessment-dialog.tsx

```diff
@@ -0,0 +1,99 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+import { css } from '@uifabric/utilities';
+import * as Markup from 'assessments/markup';
+import { BlockingDialog } from 'common/components/blocking-dialog';
+import { NewTabLink } from 'common/components/new-tab-link';
+import { Tab } from 'common/itab';
+import { NamedFC } from 'common/react/named-fc';
+import * as commonDialogStyles from 'DetailsView/components/common-dialog-styles.scss';
+import * as styles from 'DetailsView/components/target-change-dialog.scss';
+import { DefaultButton, DialogFooter, DialogType, TooltipHost } from 'office-ui-fabric-react';
+import * as React from 'react';
+export interface ChangeAssessmentDialogProps {
+    prevTab: Tab;
+    newTab: Tab;
```

Comment:
This newTab is not required since it's not being used in the component anywhere.